### ISSUE TITLE
How to check the accuracy  for every epoch?

### ISSUE THREAD

**Issue by lunasdejavu** [matterport/Mask_RCNN#1116]
Since Overfitting is an important problem, is there anyone know how to write the code to print the training accuracy and validation accuracy just like the loss function?
So that we can know it is overfitting or not by evaluating the graph of training accuracy and validation accuracy.
Sorry, I am just a beginner.
